doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+Observatory Control User Guide
+##############################
+
+.. image:: https://img.shields.io/badge/GitHub-ts_observatory_control-green.svg
+    :target: https://github.com/lsst-ts/ts_observatory_control
+.. image:: https://img.shields.io/badge/Jenkins-ts_observatory_control-green.svg
+    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_observatory_control/
+.. image:: https://img.shields.io/badge/Jira-ts_observatory_control-green.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_observatory_control
+
+The Observatory Control package provides users and developers with tools to interact with the Rubin Observatory System.
+These tools are mainly Python classes/objects that represent major groups of components and encapsulate high-level operations involving those components.
+For instance, the `ATCS` class combine functionality for all "auxiliary telescope" components of the Vera Rubin Auxiliary Telescope.
+Actions like slewing the telescope, while making sure all involved components are in `ENABLED` state and waiting until all components are in position is encapsulate in a single command;
+
+.. code:: python
+
+   from lsst.ts.observatory.control.auxtel import ATCS
+
+   atcs = ATCS()
+
+   await atcs.start_task
+
+   await atcs.slew_object(name="Alf Pav")
+
+.. A set of utilities are also provided that allow users to combine component coordination with activities like data analysis and more.
+
+In the following sections we provide some general guidance and insights in using these classes as well as a detailed description of the main operations available on each one of them.
+
+.. _user-guide-sharing-resources:
+
+Sharing resources
+=================
+
+When using multiple classes, for instance to control telescope and instrument, it is highly recommended to share some resources, like the DDS domain.
+This can be done by creating a `salobj.Domain`_ and passing it to the classes.
+
+.. _salobj.Domain: https://ts-salobj.lsst.io/py-api/lsst.ts.salobj.Domain.html#lsst.ts.salobj.Domain
+
+.. code:: python
+
+   from lsst.ts import salobj
+   from lsst.ts.observatory.control.auxtel import ATCS, LATISS
+
+   domain = salobj.Domain()
+
+   atcs = ATCS(domain=domain)
+   latiss = LATISS(domain=domain)
+
+   await atcs.start_task
+   await latiss.start_task
+
+.. _user-guide-usages-in-sal-scripts:
+
+Usages in SAL Scripts
+=====================
+
+When writing a `SAL Script`_ for the `ScriptQueue`_ it is possible to use the scripts own domain.
+In addition, it is also possible to pass the script log object so that logging from the class will also be published to SAL.
+
+.. code:: python
+
+   from lsst.ts import salobj
+   from lsst.ts.observatory.control.auxtel.atcs import ATCS
+
+
+   class MyScript(salobj.BaseScript):
+
+     def __init__(self, index):
+
+         super().__init__(index=index, descr="My script for some operation.")
+
+         self.atcs = ATCS(domain=self.domain, log=self.log)
```

Comment:
You should probably show the instantiation of the log object?

Reply:
No, it is part of the Script class already... I'll mention it in the text...